utils/create-database.js

```diff
@@ -0,0 +1,66 @@
+// utils/create-database.js
```

Comment:
All the comments in this file are not really providing any value, they are just describing _how_ you are doing what you are and not _why_, which is a far more valuable thing to provide to whoever is trying to understand and work on your code. As it is, this is clear enough to understand without having to add the comments and so they could all be removed.